src/sign-in-xplat/README.md

```diff
@@ -0,0 +1,113 @@
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+1. An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
+1. [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/#download-preview)
+1. [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+1. [.NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation)
+
+## Setup
+
+### 1. Register the web API application in your Azure Active Directory
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use the following settings for your app registration:
+
+| App registration <br/> setting | Value for this sample app                                         | Notes                                                                                                       |
+|:------------------------------:|:------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **Name**                       | `active-directory-dotnet-xplat-maui`                              | Suggested value for this sample. <br/> You can change the app name at any time.                             |
+| **Supported account types**    | **My organization only**                                          | Required for this sample. <br/> Support for the Single tenant.                                              |
+| **Platform type**              | `Mobile and desktop applications`                                 | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
+| **Redirect URIs**              | `https://login.microsoftonline.com/common/oauth2/nativeclient`    | Required value for this sample. <br/> You can change that later in your own implementation.                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box or select in the Azure portal.
+
+<details>
+   <summary>:computer: Alternative: Register the application using az-cli</summary>
+
+1. Register a new Azure AD App with a reply url
+
+   ```bash
+   AZURE_AD_APP_CLIENT_ID_XPLAT=$(az ad app create --display-name "active-directory-dotnet-xplat-maui" --reply-urls "https://login.microsoftonline.com/common/oauth2/nativeclient" --native-app true --query appId -o tsv)
+   ```
+
+</details>
+
+### 2. Configure the web app
+
+1. Open the `XPlat.csrpoj` under the the `sign-in-xplat` folder in your code editor.
```

Comment:
Paths and filenames are italicized.

Reply:
thanks for pointing this out @mmacy 


done | addressed from [70bd2](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/70bd25a7c57661c77697ab34bac2ac61ba3ca07a)

Reply:
The https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/70bd25a7c57661c77697ab34bac2ac61ba3ca07a version is close:

`1. Open the _~/sign-in-xplat/MainPage.xaml.cs_ in your code editor.`

However, we shouldn't assume the project dir resides in the user's homedir (`~`). You could use this format or something like it that doesn't include the tilde (since we don't know where their repo clone or extracted archive resides):

- Open the _sign-in-xplat/MainPage.xaml.cs_ file in your editor.

Reply:
good call @mmacy 


done | [a35d5](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/a35d5be30b798476f8c80887d9840ded74593791)